src/platform/common/utils/localize.ts

```diff
@@ -77,25 +80,28 @@ export namespace GitHubIssue {
         'Please provide a descriptive title summarizing your issue.'
     );
     export const titlePlaceholder = localize(
-        'GitHubIssue.titlePlaceholder',
+        { key: 'GitHubIssue.titlePlaceholder', comment: ['{Locked="kernel"}'] },
         'E.g.: Unable to start a local kernel session'
     );
     export const pleaseFillThisOut = localize(
         'GitHubIssue.pleaseFillThisOut',
         'Please fill this section out and delete this comment before submitting an issue!'
     );
     export const missingFields = localize(
-        'GitHubIssue.missingFields',
+        { key: 'GitHubIssue.missingFields', comment: ['{Locked="Issue"}'] },
```

Comment:
I am assuming we want to keep the uppercase `Issue` as is and not change it from language to language.